cmd/gen-release-notes/templates/minorReleaseNotes.md

```diff
@@ -1,7 +1,13 @@
 ---
 title: Change Notes
+linktitle: <!--newRelease-->
+subtitle: Minor Release
 description: Istio <!--newRelease--> release notes.
+publishdate: 2020-07-29
```

Comment:
is this hard coded?

Reply:
it's expected that the release managers change it after generating the release notes. Not sure if we can automate figuring out the release date.